ChadNaps/fun-kickoff-project
# Testing - Backend - Ping logout controller

### What this PR does (required):
- Runs a test, using supertest and mocha, on the /logout route

### Any information needed to test this feature (required):
1. Ensure you are in the `server` folder
2. Run `npm install` to ensure node packages are installed
3. Run `npm test` to run test suite
